python_modules/dagster/dagster_tests/definitions_tests/asset_reconciliation_tests/partition_scenarios.py

```diff
@@ -195,15 +202,15 @@
         unevaluated_runs=[
             run(["hourly"], partition_key=partition_key)
             for partition_key in hourly_partitions_def.get_partition_keys_in_range(
-                PartitionKeyRange(start="2013-01-06-00:00", end="2013-01-06-23:00")
+                PartitionKeyRange(start="2013-01-05-00:00", end="2013-01-05-23:00")
```

Comment:
same thing as before, just backing up the time to the start of the partitions def